jarbas/core/management/commands/companies.py

```diff
@@ -84,3 +91,7 @@ def to_email(email):
 
         except ValidationError:
             return None
+
+
+class InputDateField(rows.fields.DateField):
+    INPUT_FORMAT = '%d/%m/%Y'
```

Comment:
I'd declare this class in the beginning of the file so when you use it in the other class the reader is aware about what this field is ; )